File: python/prism_sdk/throughput_instrument_execution_integrity_inference.py
"""Lab P32 throughput inference instrument-execution integrity feature F03."""
from .instrument_execution_integrity_support import InstrumentExecutionRequest4,InstrumentExecutionCard7,InstrumentExecutionIntegrityError,manifest,qualify
FEATURE_ID="AFA-lab-P32-F03";CONTRACT_VERSION="lab-throughput_instrument_execution_integrity_inference/1.0"
def throughput_instrument_execution_integrity_inference_manifest():return manifest(feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="throughput",mode="inference")
def qualify_throughput_instrument_execution_integrity_inference(request:InstrumentExecutionRequest4)->InstrumentExecutionCard7:return qualify(request,feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="throughput",mode="inference")

File: crates/lab/src/throughput_instrument_execution_integrity_inference.rs
//! Lab P32 throughput inference instrument-execution integrity feature.
use super::instrument_execution_integrity_support::{manifest,qualify,InstrumentExecutionCard7,InstrumentExecutionIntegrityError,InstrumentExecutionRequest4};
pub const FEATURE_ID:&str="AFA-lab-P32-F03";pub const CONTRACT_VERSION:&str="lab-throughput_instrument_execution_integrity_inference/1.0";
pub fn throughput_instrument_execution_integrity_inference_manifest()->serde_json::Value{manifest(FEATURE_ID,CONTRACT_VERSION,"throughput","inference")}
pub fn qualify_throughput_instrument_execution_integrity_inference(request:&InstrumentExecutionRequest4)->Result<InstrumentExecutionCard7,InstrumentExecutionIntegrityError>{qualify(request,FEATURE_ID,CONTRACT_VERSION,"throughput","inference")}
